scss/_utilities_layout.scss

```diff
@@ -4,4 +4,14 @@
   .u-fixed-width {
     @extend %fixed-width-container;
   }
+
+  @if ($table-layout != 'fixed') {
+    .u-table-layout--fixed {
```

Comment:
I really dislike single style utilities. If the user has to apply this to each table then they might as well use `style` attribute.

Reply:
there is a subtle difference - an inline style always causes a redefinition of the rule it overrides. A conditionally generated utility overrides only if the value it sets is different from the default.